runtime/oti/j2sever.h

```diff
@@ -84,6 +80,7 @@
 #define J2SE_SHAPE_RAW	 		0x90000
 #define J2SE_SHAPE_MASK 		0xF0000
 #define J2SE_SHAPE_SHIFT		16
+#define J2SE_JAVA_SEPC_VERSIO_SHIFT 8
```

Comment:
There are a couple of typos in this name

Reply:
Fixed